Source/Quartzmin/Quartzmin.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net452</TargetFrameworks>
```

Comment:
NetStandard 2.1 should not be needed - 2.0 is perfectly supported by NetCoreApp 3.1.
However, using NetStandard 2.1 would break compatability with targets like NetCoreApp 2.1, so you might not want to do that.